xml/System.Formats.Asn1/AsnDecoder.xml

```diff
@@ -1548,17 +1730,21 @@ This method does not determine if the string used only characters defined by the
         <param name="bytesConsumed">When this method returns, the total number of bytes for the encoded value.
                This parameter is treated as uninitialized.</param>
         <param name="expectedTag">The tag to check for before reading, or <see langword="null" /> for the default tag (Universal 2).</param>
-        <summary>Attempts to read an Integer value from <paramref name="source" /> with a specified tag under the specified encoding rules as an unsigned 64-bit value.</summary>
+        <summary>Attempts to read an integer value from <paramref name="source" /> with a specified tag under the specified encoding rules as an unsigned 64-bit value.</summary>
```

Comment:
I believe I used a capital-I Integer here because it's the ASN.1 datatype name, not just the concept of an integer.

If lowercase makes more sense there are a lot of other instances that should be changed (the `<returns>` for this one, and then this should also exist in all of the other (ASN.1-)Integer-based methods.

Mainly I'm calling it out as "I don't think this one should be different".